src/components/Footer.tsx

```diff
@@ -0,0 +1,90 @@
+import React, { useContext } from 'react';
+import cn from 'classnames';
+import { Status } from '../services/EnumStatusFilter';
+import { TodoContext } from './TodoContext';
+
+export const Footer: React.FC = () => {
+  const {
+    deleteAllCompleted,
+    count,
+    selectTodoFilteredList,
+    setSelectTodoFilteredList,
+    todos,
+  } = useContext(TodoContext);
+
+  return (
+    <footer className="footer">
+      <span className="todo-count" data-cy="todosCounter">
+        {`${count} items left`}
+      </span>
+
+      <ul
+        className="filters"
+        data-cy="todosFilter"
+      >
+        <li>
+          <a
+            data-value={Status.ALL}
+            href="#/"
+            className={
+              cn({
+                selected: selectTodoFilteredList === Status.ALL,
+              })
+            }
```

Comment:
```suggestion
            className={cn({ selected: selectTodoFilteredList === Status.ALL })}
```

new lines here are redundant